src/index.html

```diff
@@ -11,20 +11,16 @@ <h1>Leaderboard</h1>
         <div class="board-container">
             <h3>Recent score <button class="re-button">Refresh</button></h3>
             <ul class="board">
-                <li>Name: 200</li>
-                <li>Name: 100</li>
-                <li>Name: 95</li>
-                <li>Name: 33</li>
-                <li>Name: 800</li>
-                <li>Name: 10</li>
             </ul>
         </div>
-        <div class="submit">
+        <div >
+        <form class="submit" >
             <h3>Add your score</h3>
-            <input placeholder="Your Name">
-            <input placeholder="Your score">
-            <button class="submit-but">Submit</button>
-        </div>
+            <input class="name" type="text" placeholder="Your Name">
+            <input class="score" placeholder="Your score">
```

Comment:
- [ ] [OPTIONAL] It would be nice if you implement `<input type="number" class="score" placeholder="Your score">` instead of  `<input class="score" placeholder="Your score">` 👍🏼

Reply:
Thank you for you suggestion